consensus/core/src/mass/mod.rs

```diff
@@ -66,3 +72,253 @@ pub fn transaction_output_estimated_serialized_size(output: &TransactionOutput)
     size += output.script_public_key.script().len() as u64;
     size
 }
+
+// TODO (aspect) - review and potentially merge this with the new MassCalculator currently located in the wallet core
```

Comment:
Can this TODO be updated or removed?

Reply:
changed the comment noting that consensus `MassCalculator` operates on signed transactions and to use `kaspa_wallet_core::tx::mass::MassCalculator` if the developer needs to calculate mass on unsigned transactions.